docs/app/insights.md

```diff
@@ -1,81 +1,199 @@
+- **Suffix** - Adds text after the metric
+- **Conditional Styles** - Changes color of number displayed if it is `>`, `>=`, `<`, `<=`, `==` (equal to), or `!=`
+  (not equal to) to some defined value
+
+### Time Series Overview
+
+The Time Series panel allows you to build graphs and see how data changes over time.
+
+::: tip Note In order to use this panel, your collection will need a datetime field. :::
+
+![image of Time Series panel options](image.jpg)
+
+- **Collection** - Selects the collection you want to analyze
+- **Color** - Sets color of your graph curve
+- **Group Aggregation** - Selects the type of aggregation to perform
+- **Group Precision** - Sets the time interval to aggregate by _(days, weeks, months etc.)_
+- **Date Field** - Selects the datetime field used along the x-axis.
+- **Date Range** - Selects the time range from which data should be displayed
+- **Value Field** - Sets the field you wish to aggregate
+- **Value Decimals** - Changes the number of decimals displayed on the y-axis
+- **Min Value** - Sets minimum value displayed on y-axis
+- **Max Value** - Sets maximum value displayed on y-axis
+- **Curve Type** - Sets curve to be shown as smooth, straight or stepline
+- **Fill Type** - Sets fill type under curve to gradient, solid, or none
+- **Filter** - Allows filtering of items considered for analysis based on logical conditions
+- **Show X-axis** - Toggles display of time along X-axis
+- **Show Y-axis** - Toggles display of numeric values along Y-axis
+
+### Advanced Filtering
+
+The filtering options are infinitely customizable. However, they will be limited by the collections, relations, and
+information stored in your database.
+
+#### Nested Filters
+
+Sometimes, you will need to run filters that require information from two or more collections. Here might be a few
+examples:
+
+- Highest paid authors that wrote 5 or more books (uses `authors` and `books` collections)
+- Products with less than 50 sales monthly (uses `products` and `orders` collections)
+- Blog Posts with the highest number of social shares (uses `blog` and `social share metadata` collections )
+
+Nested filters limit results of one collection by the values of fields from a related collection. Let's look at how to
+create a nested filter:
+
+![Image of a nested filter with numbers](image.jpg)
+
+1. Create a filter
+2. Select a foreign key field
+3. Create a new filter
+4. Nest it under the first filter by clicking and dragging the lower filter to the right
+5. Select the filters to apply on this collection
+
+::: tip Foreign Key A _foreign key field_ is a field that references the primary key (often times this is the `ID`)
+field of another collection. ::: ::: tip How to filter against other tables Once a foreign key is selected on a filter,
+any directly nested filters will take on the other fields from that collection. :::
+
+## Tutorial Video
+
+Learn to make analytics dashboards with meaningful insights panels.
```

Comment:
If we do a formal video, let's make sure to create 2-3 really nice dashboards for the example. The ones in our demos have obviously fake data and should be polished up a bit.

Reply:
@benhaynes  for the first "real" PR, I'm going to cut this entirely then add it back later... But wanted to note it now.
Over the next few weeks, I'd like to help develop an "insanely swag" practice database.
... Something generic enough for beginners, robust enough for "pros" to take respectably, and a data model that covers several major use cases.... Def need some time to get my feet wet, though.